<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLicense">
    <property name="title">
     <string>License</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionReadMe"/>
    <addaction name="actionContactMe"/>
   </widget>
   <addaction name="menuLicense"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionReadMe">
   <property name="text">
    <string>Read me</string>
   </property>
  </action>
  <action name="actionContactMe">
   <property name="text">
    <string>Contact me</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
